Expensify/App
# [Chat] Add drag and drop handlers to web/desktop

cc @thienlnam @roryabraham for extra eyes since y'all reviewed a similar one to this

### Fixed Issues
Fixes https://github.com/Expensify/Expensify/issues/141090

### Tests
1. Drag various files from the finder into the text input
1. Verify the text input is highlighted when a file is over it
1. Verify that you can drag and drop images, PDFs, zip files, and a preview opens
1. Verify you can upload the file type if supported

### Screenshots
![Screen Shot 2020-12-09 at 3 25 35 PM](https://user-images.githubusercontent.com/32969087/101709201-2301dd80-3a33-11eb-9924-d852368a9c8d.png)
![Screen Shot 2020-12-09 at 3 25 37 PM](https://user-images.githubusercontent.com/32969087/101709206-25643780-3a33-11eb-9912-3cca4afbdc3e.png)
![Screen Shot 2020-12-09 at 3 25 39 PM](https://user-images.githubusercontent.com/32969087/101709247-344aea00-3a33-11eb-82c0-0568bf67c879.png)
